db/seeds.rb

```diff
@@ -12,103 +12,120 @@
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/LegOfGriffin.jpg'))
 
 leg_of_pegasus = Item.create(title: "Leg of Pegasus",
                              description: "This dish will send you galloping off to tasty bliss",
                              price: 9.50,
                              status: "retired",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/LegOfPegasus.jpg'))
 
 hydra          = Item.create(title: "Hydra Vodka",
                              description: "Careful with this one! We serve it so virulent your friends may appear to have many heads",
                              price: 6.00,
                              status: "active",
-                             categories: [drinks_cat])
+                             categories: [drinks_cat],
+                             image: File.open('public/HydraVodka.jpg'))
 
 rainbow_pie          = Item.create(title: "Ogre Pie",
                              description: "We hope your appetite is as voracious as this dish's namesake",
                              price: 5.50,
                              status: "active",
-                             categories: [dessert_cat])
+                             categories: [dessert_cat],
+                             image: File.open('public/OgrePie.jpg'))
 
 centaur        = Item.create(title: "Centaur Sushi",
                              description: "Don't get 'carried away' with these bite size nibblers",
                              price: 18.00,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/CentaurSushi.jpg'))
 
 cerberus       = Item.create(title: "Cerberus Chow",
                              description: "A snack popular with the hound guard of underworld",
                              price: 14.50,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/CerberusChow.jpg'))
 
 harpy          = Item.create(title: "BBQ Harpy Wings",
                              description: "The wings of this fluttery creature--known for its thievery--are a steal themselves",
                              price: 6.50,
                              status: "active",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/BBQHarpyWings.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.